src/render.ts

```diff
@@ -172,7 +172,13 @@ export function render(s: State): void {
 }
 
 export function updateBounds(s: State): void {
+  const bounds = s.dom.elements.wrap.getBoundingClientRect();
+  const container = s.dom.elements.container;
+  container.style.width = (Math.floor(bounds.width / 8) * 8).toString() + 'px';
+  container.style.height = (Math.floor(bounds.height / 8) * 8).toString() + 'px';
```

Comment:
for non-square 3d boards, i believe what this should really do is round width to 8px steps, and for height, preserve the aspect ratio.

Reply:
Yeah, you're right, changed it. Though that also doesn't seem to make it pixel perfect. But on 3D boards it's at least not very noticeable and should be good enough for now.